src/applications/personalization/view-dependents/components/ViewDependentsLayout.jsx

```diff
@@ -1,8 +1,9 @@
 import React, { Component } from 'react';
 
 import Breadcrumbs from '@department-of-veterans-affairs/formation-react/Breadcrumbs';
-import ViewDependentsList from './ViewDependentsList';
-import ViewDependentsSidebar from './ViewDependentsSidebar';
+import ViewDepenentsList from './ViewDependentsList/ViewDependentsList';
+import ViewDepenentsSidebar from './ViewDependentsSidebar/ViewDependentsSidebar';
```

Comment:
This should be `ViewDependentsList` and `ViewDependentsSidebar`

Reply:
updated.